wpa_passphrase: add page (#3301)
tldr-pages/tldr: pages/linux/wpa_passphrase.md

# wpa_passphrase

> > Generate a WPA PSK from an ASCII passphrase for a SSID.

Reviewer: ```suggestion
> Generate a WPA-PSK key from an ASCII passphrase for a given SSID.
```

> - Display WPA PSK for a SSID reading passphrase from stdin:

Reviewer: ```suggestion
- Compute and display the WPA-PSK key for a given SSID reading the passphrase from stdin:
```
I think only saying "display" could be interpreted as "the key is already stored somewhere". Let's say "*compute* and display" instead.

> `wpa_passphrase {{ssid}}`

Reviewer: ```suggestion
`wpa_passphrase {{SSID}}`
```

> - Display WPA PSK for a SSID with a specified passphrase:

Reviewer: ```suggestion
- Compute and display the WPA-PSK key for a given SSID with specifying the passphrase as an argument:
```
Same as the previous description basically.

> `wpa_passphrase {{ssid}} {{passphrase}}`

Reviewer: ```suggestion
`wpa_passphrase {{SSID}} {{passphrase}}`
```
